### PR TITLE
Chat AI Demo: Add model to params, fix import in Vue demo

### PR DESCRIPTION
**Pull request by Zedwag**

### CODE REVIEW

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/React/App.tsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ async function getAIResponse(messages) {
     temperature: 0.7,
   };
 
-  const response = await chatService.chat.completions.create(params);
+  const response = await chatService.chat.completions.create(params as any);
```
Comment (anna-shakhova) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/React/App.tsx]: I think we should avoid 'any' and assertions as untipatterns in our demos
We can define params using required type, add examples with some artificial type - you can change it for the required one 

const params: ChatServiceCreateParams = {
    messages,
    max_tokens: 1000,
    temperature: 0.7,
  };

Comment (Zedwag) [reply apps/demos/Demos/Chat/AIAndChatbotIntegration/React/App.tsx]: i did it to avoid required `model` passing
image
but ok i guess it's ok to use the one we use in service config

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/React/App.tsx [hunk 2]
```diff
@@ -31,6 +31,7 @@ const chatService = new AzureOpenAI(AzureOpenAIConfig);
 async function getAIResponse(messages) {
   const params = {
     messages,
+    model: AzureOpenAIConfig.deployment,
```
Comment (marker-dao) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/React/App.tsx:34]: Why did you add that?

Comment (ksercs) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/React/App.tsx:34]: don't we want to apply this in all frameworks to get rid of ts warnings everywhere?

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/App.vue [hunk 3]
```diff
@@ -76,6 +76,8 @@ const copyButtonIcon = ref('copy');
 
 onBeforeMount(() => {
   loadMessages(dictionary);
+
+  chatService = new AzureOpenAI(AzureOpenAIConfig);
```
Comment (ksercs) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/App.vue]: I've checked this change on az-server and seems like it does not resolve the issue.
However, replacing current import with `import openai from 'openai';` helps.
So I suggest you to revert this change but update the import

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/React/App.tsx (hunk 2 above)
Comment (marker-dao) [reply apps/demos/Demos/Chat/AIAndChatbotIntegration/React/App.tsx:34]: okay, got it (cuz ts)

Comment (marker-dao) [reply apps/demos/Demos/Chat/AIAndChatbotIntegration/React/App.tsx:34]: [b83ec79eabe4b3dfb5b357bd0ea9b4bfe39a0859]

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/App.vue (hunk 3 above)
Comment (marker-dao) [reply apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/App.vue]: [b83ec79eabe4b3dfb5b357bd0ea9b4bfe39a0859]

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/App.vue [hunk 4]
```diff
@@ -53,10 +53,9 @@ import { ref, onBeforeMount } from 'vue';
 import DxChat from 'devextreme-vue/chat';
 import DxButton from 'devextreme-vue/button';
 import { loadMessages } from 'devextreme/localization';
-import { AzureOpenAI } from 'openai';
+import openai from 'openai';
 import {
   dictionary,
-  store,
```
Comment (ksercs) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/App.vue:59]: Is this change intentional?

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/data.ts [hunk 5]
```diff
@@ -26,8 +26,7 @@ export const CHAT_DISABLED_CLASS = 'dx-chat-disabled';
 export const ALERT_TIMEOUT = 1000 * 60;
 
 export const user = {
-  id: 'c94c0e76-fb49-4b9b-8f07-9f93ed93b4f3',
-  name: 'John Doe',
+  id: 'user',
```
Comment (ksercs) [inline apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/data.ts:29]: For what?

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/App.vue (hunk 4 above)
Comment (marker-dao) [reply apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/App.vue:59]: Yes, store is not used anywhere, its import was an oversight.

File: apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/data.ts (hunk 5 above)
Comment (marker-dao) [reply apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/data.ts:29]: Look at the other demos (jQuery for example): there's different data there, it just wasn't copied correctly.

**Review by ksercs — APPROVED** [review]
